bots/image-refresh

```diff
@@ -62,23 +62,19 @@ TRIGGERS = {
     "openshift": [
         "container/kubernetes",
         "verify/fedora-28",
-        "verify/rhel-7-5"
+        "verify/rhel-7-6"
     ],
     "ipa": [
         "verify/fedora-28",
-        "verify/rhel-7-5",
-        "verify/rhel-7-5-distropkg",
-        "verify/ubuntu-1604",
+        "verify/rhel-7-6",
+        "verify/rhel-7-6-distropkg",
+        "verify/ubuntu-1804",
```

Comment:
Isn't this ubuntu change unrelated to this patch?

Reply:
Ah sorry, it is; probably should have been a separate commit. ubuntu-1604 doesn't even exist any more.

OK to mention this in  the commit message when merging, or want me to split it out?

Reply:
Split out into separate commit, thanks for catching. Test results copied with push-rewrite.